supplier_app/tests/test_view.py

```diff
@@ -739,15 +748,24 @@ def test_taxpayer_detail_view_as_supplier_dont_show_approve_denied_btn(self):
 
 class TestEditTaxPayerInfo(TestCase):
     def setUp(self):
-        self.factory = RequestFactory()
         self.client = Client()
+        self.client2 = Client()
 
-        self.ap_user = User.objects.create_user(email='ap@eventbrite.com')
-        self.ap_user.groups.add(Group.objects.get(name='ap_admin'))
-        self.user_with_social_evb1 = UserFactory(email='nahuel')
-        self.user_with_social_evb1.groups.add(Group.objects.get(name='supplier'))
+        self.ap_group = Group.objects.get(name="ap_admin")
+        self.ap_user = UserFactory(email='ap@eventbrite.com')
+        self.ap_user.groups.add(self.ap_group)
+        self.client.force_login(self.ap_user)
+
+        self.supplier_group = Group.objects.get(name="supplier")
+        self.supplier_user = UserFactory(email='nahuelSupplier@gmail.com')
+        self.supplier_user.groups.add(self.supplier_group)
+        self.client2.force_login(self.supplier_user)
 
         self.taxpayer = TaxPayerArgentinaFactory()
+        self.comapnyuserpermission = CompanyUserPermissionFactory(
```

Comment:
Please replace `self.comapnyuserpermission` by `self.company_user_permission`. Also consider thinking if we are talking about company permission, or user inside company permission, or company permission, to be clear about what are we talking about.

Reply:
CompanyUserPermission is the name of the table that we created at first. But now we are using the table of Django's permissions.